rel/common/launch.sh

```diff
@@ -184,6 +184,11 @@ case "$1" in
 
         HEART_COMMAND="$RUNNER_BASE_DIR/bin/$SCRIPT start"
         export HEART_COMMAND
+
+        # Uncomment the below two lines ONLY if you face leo_(manager|storage|gateway) restarted frequently
+        # in order to dig down further for identifying the root cause.
```

Comment:
I've felt almost all of people do not know `ERL_CRASH_DUMP_SECONDS=-1`, it would be good to describe `ERL_CRASH_DUMP_SECONDS` as below:

`ERL_CRASH_DUMP_SECONDS=-1` means setting the environment variable to a negative value does not reboot the runtime system until the crash dump file is completely written.